README.md

```diff
@@ -1,55 +1,149 @@
 
-2. Install all the dependencies using the following command
+To get started with the CLI is very simple
 
-   ```bash
-   npm install
-   ```
-   
-3. Create a symlink of the repository in the global folder
+1.  You can choose to clone or download this repository.  
+    To clone this repository, the command is  
 
-   ```bash
-   sudo npm link
-   ```
+    ```bash
+    git clone https://github.com/dmahajan980/covid19.git
+    ```
 
-## USAGE
+2.   Now you need to install the dependencies. To do so, run the command
+      ```bash
+      npm install
+      ```
 
-The CLI will return the latest global statistics about COVID-19 in a tabular format.
+3.   The last step is to add a symbolic link to the global folder.  
+    If you are on Windows, start `cmd.exe` as Administrator and run the command
+      ```bash
+      npm link
+      ```
+      If you are on macOS/Linux, the command is
+      ```bash
+      sudo npm link
+      ```
 
-   ```bash
-   covid19 stats
-   ```
+And we are ready to go!
 
-The CLI can also return the COVID-19 statistics of a country, if specified, using one of the following commands.
+## Usage
 
-   ```bash
-   covid19 stats --country <countryName>
-   ```
+This CLI is very easy to use.
 
-   ```bash
-   covid19 stats -c <countryName>
-   ```
+*   **covid19**  
+    This is the name of the package that is used to invoke the CLI.  
+    Flags are available for this command.  
 
-The supported formats for `countryName` are ISO2, ISO3 or a single-word name.
+    <table style="height: 5px; width: 600px;" border="black">
 
-## CONTRIBUTING
+    <tbody>
+
+    <tr style="border-color: black;">
+
+    <td style="width: 150px; text-align: center; border-color: black;"><span style="text-decoration: underline;">flag</span></td>
+
+    <td style="width: 150px; text-align: center; border-color: black;"><span style="text-decoration: underline;">alias</span></td>
+
+    <td style="width: 300px; text-align: center;  border-color: black;"><span style="text-decoration: underline;">description</span></td>
+
+    </tr>
+
+    <tr>
+
+    <td style="width: 150px; text-align: center; border-color: black;">--version</td>
+
+    <td style="width: 150px; text-align: center; border-color: black;">-v</td>
+
+    <td style="width: 300px; text-align: center; border-color: black;">Display the current version of the program</td>
+
+    </tr>
+
+    </tbody>
+
+    </table>
+    <br/>
+
+    Example:
+    ```bash
+    covid19 -v
+    ```
+    OR
+    ```bash
+    covid19 --version
+    ```
+
+*   **covid19 stats**  
+    This command returns the statistics of CoVID 19 all over the world.
+
+    Example:
+    ```bash
+    covid19 stats
+    ```
+
+    Flags are available for this command.  
+
+    <table style="height: 5px; width: 600px;" border="black">
+
+    <tbody>
+
+    <tr style="border-color: black;">
+
+    <td style="width: 150px; text-align: center; border-color: black;"><span style="text-decoration: underline;">flag</span></td>
```

Comment:
You should probably remove the `width` from the `style` attribute for the first column since this doesn't fit in the whole command (see the screenshot below).

![1](https://user-images.githubusercontent.com/37256767/96143249-8b937900-0f20-11eb-9b0e-457411801f1d.png)

In general, a better way to avoid these scenarios is to use the [markdown table syntax](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/organizing-information-with-tables) instead of the HTML tags, especially when you want to use a table in "README" or some other markdown file.

Reply:
Thank you for this. I was not well versed with markdown.
But I will definitely use this syntax for this project and all my future projects.